src/plugins/saved_objects_finder/public/finder/index.tsx

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React from 'react';
+import type { SavedObjectFinderProps } from './saved_object_finder';
+
+const LazySavedObjectFinder = React.lazy(() => import('./saved_object_finder'));
+const SavedObjectFinder = (props: SavedObjectFinderProps) => (
+  <React.Suspense fallback={null}>
```

Comment:
How about showing a deferred spinner like this? 
https://github.com/elastic/kibana/blob/a99080dbb89b6e9b5ea3583ee84ed911eef8d6fb/src/plugins/discover/public/plugin.tsx#L244-L250

Reply:
Sounds good to me!